woocommerce/woocommerce-ios
# [Mobile Payments] Add isLive and isInTestMode to PaymentGatewayAccount entity

Partially addresses #4428 

@joshheald @jaclync - if either of you could take a look, that would be great - just one of y'all are needed of course, but both of y'all are welcome

- This PR is the next step in detecting that a store has a live account in test mode (so that we don't attempt to connect to readers, which will fail for live sites in test mode)
- This PR adds the isLive (defaulted to true) and isInTestMode (defaulted to false) to core data and rolls the model version
- The next (and last) PR will populate the new Bools from the remote and finish implementing isWCPayInTestModeWithLiveStripeAccount in CardPresentPaymentsOnboardingUseCase

To test:
- Ensure unit tests pass, especially the new migration test

Update release notes:
